### ISSUE TITLE
Multline strings are not highlighted properly

### ISSUE THREAD

**Issue by rdburns** [paul-donahue/systemrdl-mode#1]
Multiline strings are only highlighted as strings on the first line.

I see that the mode is derived from the c-mode. In C multiline strings have a backslash before the newline, but this isn't the case for SystemRDL, you're allowed to just run a string across multiple lines.

**Comment by paul-donahue** [comment]
Yes, this isn't the only weirdness with deriving from c-mode.  Unfortunately, I haven't had time[^1] to develop a ground-up mode that doesn't derive from one of the other modes such as c-mode.  I tried [prog-mode](https://www.emacswiki.org/emacs/ProgMode) and [smie](https://www.gnu.org/software/emacs/manual/html_node/elisp/SMIE.html) but ran into some difficulties that I can't remember 2 years later.  As much as I'd like to fix this, I don't anticipate doing so anytime soon.  I certainly welcome PRs, though.

[^1]: or significant emacs lisp expertise, though gaining that is really just a matter of time

**Comment by rdburns** [comment]
It's ok - I also found this one:
https://github.com/luisgutz/emacs-system-rdl

**Comment by paul-donahue** [comment]
That one didn't exist when I started and I only wrote this because I couldn't find anything else.  That one derives from sh-mode which is one of the base modes that I didn't try (because it's not obvious to me that SystemRDL and Bourne shell are similar).  But if it works then great.
